ui-meta/slurm.yml

```diff
@@ -0,0 +1,17 @@
+name: "slurm"
+label: "Slurm"
+description: "Slurm OpenHPC cluster."
+
+parameters:
```

Comment:
We also need to expose user definitions, somehow.

Reply:
See discussion on the Slack channel